core/chains/cosmos/denom/denom.go

```diff
@@ -0,0 +1,38 @@
+package denom
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func init() {
+	for _, d := range []struct {
+		denom    string
+		decimals int64
+	}{
+		{"atom", 0},
+		{"matom", 3},
+		{"uatom", 6},
+	} {
+		dec := sdk.NewDecWithPrec(1, d.decimals)
+		if err := sdk.RegisterDenom(d.denom, dec); err != nil {
+			panic(fmt.Errorf("failed to register denomination %q: %w", d.denom, err))
+		}
+	}
+}
+
+// ConvertToAtom is a helper for converting to atom.
+func ConvertToAtom(coin sdk.Coin) (sdk.DecCoin, error) {
```

Comment:
Would it make sense to name these for the type, rather than just 'Convert*'?
```suggestion
func CoinToAtom(coin sdk.Coin) (sdk.DecCoin, error) {
```